tasks/oss.yml

```diff
@@ -34,7 +34,7 @@
         path: "{{ oss_path }}"
         state: unmounted
       when:
-        - oss_dev is defined
+        - oss_dev_is_defined
```

Comment:
Could have added the condition to the block, but no problem.

Reply:
Thanks, have addressed it.